src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,291 @@
+﻿/* @internal */
+namespace ts.codefix {
+    const nodeModulesDir = "node_modules";
+
+    registerCodeFix({
+        errorCodes: [Diagnostics.Cannot_find_name_0.code],
+        getCodeActions: (context: CodeFixContext) => {
+            const sourceFile = context.sourceFile;
+            const checker = context.program.getTypeChecker();
+            const allFiles = context.program.getSourceFiles();
+            const useCaseSensitiveFileNames = context.host.useCaseSensitiveFileNames ? context.host.useCaseSensitiveFileNames() : false;
+
+            const token = getTokenAtPosition(sourceFile, context.span.start);
+            const name = token.getText();
+            const allActions: CodeAction[] = [];
+
+            // Get existing ImportDeclarations from the source file
+            const imports: ImportDeclaration[] = [];
+            if (sourceFile.statements) {
+                for (const statement of sourceFile.statements) {
+                    if (statement.kind === SyntaxKind.ImportDeclaration) {
+                        imports.push(<ImportDeclaration>statement);
+                    }
+                }
+            }
+
+            // Check if a matching symbol is exported by any ambient modules that has been declared
+            const ambientModules = checker.getAmbientModules();
+            for (const moduleSymbol of ambientModules) {
+                const exports = checker.getExportsOfModule(moduleSymbol) || [];
+                for (const exported of exports) {
+                    if (exported.name === name) {
+                        allActions.push(getCodeActionForImport(removeQuotes(moduleSymbol.getName())));
+                    }
+                }
+            }
+
+            // Check if a matching symbol is exported by any files known to the compiler
+            for (const file of allFiles) {
+                const exports = file.symbol && file.symbol.exports;
+                if (exports) {
+                    for (const exported in exports) {

```

Comment:
actually this is not accurate, you want to use checker.ts::getExportsOfModule here to make sure `export * from "module"` are reflected here.